DATA/common/setenv_calib.sh

```diff
@@ -24,6 +24,7 @@ if has_detector_calib TRD && has_detectors ITS TPC TRD && has_detector_matching
 if has_detector_calib EMC && has_detector_reco EMC; then CAN_DO_CALIB_EMC_BADCHANNELCALIB=1; CAN_DO_CALIB_EMC_TIMECALIB=1; else CAN_DO_CALIB_EMC_BADCHANNELCALIB=0; CAN_DO_CALIB_EMC_TIMECALIB=0; fi
 if has_detector_calib PHS && has_detector_reco PHS; then CAN_DO_CALIB_PHS_ENERGYCALIB=0; CAN_DO_CALIB_PHS_BADMAPCALIB=1; CAN_DO_CALIB_PHS_TURNONCALIB=1; CAN_DO_CALIB_PHS_RUNBYRUNCALIB=1; CAN_DO_CALIB_PHS_L1PHASE=1; else CAN_DO_CALIB_PHS_ENERGYCALIB=0; CAN_DO_CALIB_PHS_BADMAPCALIB=0; CAN_DO_CALIB_PHS_TURNONCALIB=0; CAN_DO_CALIB_PHS_RUNBYRUNCALIB=0; CAN_DO_CALIB_PHS_L1PHASE=0; fi
 if has_detector_calib CPV && has_detector_reco CPV; then CAN_DO_CALIB_CPV_GAIN=1; else CAN_DO_CALIB_CPV_GAIN=0; fi
+# if has_detector_calib FT0; then CAN_DO_CALIB_FT0_TIMEOFFSET=1; else CAN_DO_CALIB_FT0_TIMEOFFSET=0; fi
```

Comment:
Hi @afurs the check actually fails because you commented out this line here:
```
/sw/slc7_x86-64/O2DPG/707-slc7_x86-64-local1/DATA/common/setenv_calib.sh: line 122: CAN_DO_CALIB_FT0_TIMEOFFSET: unbound variable
```
`CAN_DO_CALIB_FT0_TIMEOFFSET` must be defined. Can you uncomment it please? And then, if you want to disable the calibration always you should hard-code
`CALIB_FT0_TIMEOFFSET=0`  somewhere around line 143

Reply:
Ah, sorry, my bad... fixing now